### DISCUSSION TITLE
Alire can't decide system when "get-ing" eagle_lander demo

### DISCUSSION THREAD

**Discussion by RobG9876** [alire-project/alire#1191] [Q&A] [answered]
As a newby to Alire, but not to Ada or GNAT-CE/GPS, I installed Alire on my WIndows10 laptop. As an additional result it installed MSYS2. I have no intention whatsoever to use MSYS, just want an Ada/Windows10 development environment.
Skipping the "Hello, World" demo I started with Eagle_Lander, mainly because that is a Gtkada app. (I have already created numerous Gtkada apps for private use.)
However, "alr get eagle_lander" resulted in:
>>
Note: Deploying gtkada=21.0.0...
-=O=- # # # #
Note: Running post_fetch actions for gtkada=21.0.0...
configure: loading site script /usr/local/etc/config.site
checking build system type... ./config.guess: unable to guess system type
<<
Note that from the MSYS2 environment that was installed by Alire, if I do a ./doinstall for Gtkada the script fails because it can't find a bin directory with Gtk.

What to do to get Gtkada installed?
PS It is unclear to me why Alire doesn't fetch the 22.0.0 version from Gtihub

**Comment by mosteo** [comment, marked as answer]
Hi,

you can disable msys2 on Windows with `alr config --set --global msys2.do_not_install true` 

As for the gtkada problems, it seems the gtkada packaging is having [this (or similar)](https://github.com/alire-project/alire-index/pull/572/files#diff-8231323d0a4ab9710d43ed22d27a122edc644e8d93a7e23331b20195cb93b319) issue on Windows. A new release fixing it is needed from the maintainer (which is listed as @Fabien-Chouteau).
